nestadia-wgpu/src/main.rs

```diff
@@ -565,6 +583,30 @@ fn main() {
     let mut save_path = path.clone();
     save_path.set_extension("sav");
 
+    // Create the audio device
+    let (_audio_stream, audio_stream_handle) = OutputStream::try_default().unwrap();
+    let audio_sink = Sink::try_new(&audio_stream_handle).unwrap();
```

Comment:
1. We should handle the case when there is no audio device available instead of crashing with unwrap().
2. `_audio_stream` should be part of the `State` to be sure it's not dropped. This could cause issues later on.

The way I see it would work would be to create a struct containing both values and have an `Option<AudioHandler>`. If the device fetching fails, put None instead.


Another note I don't know where to write is that we should put `audio` as an optionnal (default) feature. The reason is that there are performance issue with this part right now and it's not required for most other part of the system to work. Also, since it doesn't work on my system for now, it would allow me to work regardless of that.